doc/_templates/layout.html

```diff
@@ -0,0 +1,4 @@
+{% extends "!layout.html" %}
+{% block extrahead %}
+    <link href="{{ pathto("_static/custom.css", True) }}" rel="stylesheet" type="text/css">
```

Comment:
Just noticed, does this path need `css/` in it?
```suggestion
    <link href="{{ pathto("_static/css/custom.css", True) }}" rel="stylesheet" type="text/css">
```